lib/active_model_serializers/adapter/base.rb

```diff
@@ -54,25 +54,24 @@ def include_meta(json)
         json
       end
 
-      def default_key_transform
-        :unaltered
-      end
+      class << self
+        def default_transform
```

Comment:
More docs than mentioning that there are default transforms specified for each adapter? https://github.com/remear/active_model_serializers/blob/transforms/docs/general/key_transform.md#adapter-default

Which reminds me, I need to rename that md file.